ivy/functional/frontends/tensorflow/keras/activations.py

```diff
@@ -111,6 +111,44 @@ def selu(x):
     return ivy.selu(x)
 
 
+@with_supported_dtypes(
+    {"2.13.0 and below": ("float16", "float32", "float64")},
+    "tensorflow",
+)
+def serialize(func, custom_objects=None):
```

Comment:
we need to adhere the Tensorflow frontend names for arguments it was in documentation (activation, use_legacy_format=False) you could have a look here (https://www.tensorflow.org/api_docs/python/tf/keras/activations/serialize )